lib/getOriginalConstructor/index.js

```diff
@@ -28,6 +28,11 @@ var getOriginalConstructor = function getOriginalConstructor (constructor) {
         iframe = document.createElement('iframe');
         iframe.src = 'about:blank';
         iframe.style.display = 'none';
+        iframe.height = '0';
+        iframe.width = '0';
+        iframe.tabIndex = '-1';
+        iframe.title = 'empty';
```

Comment:
Wouldn't this give the iframe the title of "empty" instead of undefined? I think we should either leave it undefined or set it as:`iframe.title = ''`

Reply:
I'm surprised this is an issue. I'd have thought the iframe would be deleted after use so this wouldn't matter.

Reply:
@sirugh makes a valid point, that's probably the better fix for this, since it was only ever intended to be used as a handle to grab a pristine copy of a prototype. I think the reason I left the iframe instance lingering around was in case people were using this function frequently, so it wouldn't have to create/append/fetch-prototype/remove over and over.